docs/generate/flags.md

```diff
@@ -90,7 +90,25 @@ Those flags are temporary and will be removed in the future. Those flags are her
 | `--keep-version-file`                                                                    | Whether you want to override the current version file in your package or not. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                              |
 | `--no-namespace-folders`                                                                 | Specify if you don't want pkgutil-style namespace folders. See [here][namespace_folders] for more information on pkgutil namespace folders. Defaults to `false`.                                                                                                                                                                                                                                                                                                |
 | `--credential-default-policy-type=BearerTokenCredentialPolicy\|AzureKeyCredentialPolicy` | Specify the default credential policy (authentication policy) for your client. Use in conjunction with `--add-credential`. Currently only supports [`BearerTokenCredentialPolicy`][bearer_token_credential_policy] and [`AzureKeyCredentialPolicy`][azure_key_credential_policy]. Default value is `BearerTokenCredentialPolicy`. `--credential-scopes` is tied with `BearerTokenCredentialPolicy`, do not pass them in if you want `AzureKeyCredentialPolicy`. |
-| `--credential-key-header-name=NAME`                                                      | The name of the header which will pass the credential. Use if you have `--credential-default-policy-type` set to [`AzureKeyCredentialPolicy`][azure_key_credential_policy]. For example, if generating cognitive services code, you might use `--credential-key-header-name=Ocp-Apim-Subscription-Key`.                                                                                                                                                         |
+| `--credential-key-header-name=NAME`                                                      | The name of the header which will pass the credential. Use if you have `--credential-default-policy-type` set to [`AzureKeyCredentialPolicy`][azure_key_credential_policy]. For example, if generating cognitive services code, you might use `--credential-key-header-name=Ocp-Apim-Subscription-Key`.|
```

Comment:
Should you document that one now that autorest support authentication built-in? Do python gen not respect it yet?